models/service_helpers.go

```diff
@@ -55,6 +55,27 @@ func checkServiceUniqueName(q *reform.Querier, name string) error {
 	}
 }
 
+func validateDBConnectionOptions(socket, host *string, port *uint16) error {
+	if host == nil && socket == nil {
+		return status.Error(codes.InvalidArgument, "Neither socket nor address passed")
```

Comment:
Error messages shouldn't be capitalized

Reply:
They should be for gRPC errors: https://jira.percona.com/browse/PMM-4695